integration/src/main/java/org/exoplatform/answer/ext/impl/AnswersSpaceActivityPublisher.java

```diff
@@ -174,6 +174,9 @@ public void moveQuestions(List<String> questions, String catId) {
           ExoSocialActivity oldActivity = activityM.getActivity(activityId);
           activityM.deleteActivity(oldActivity);
           Identity userIdentity = identityM.getOrCreateIdentity(OrganizationIdentityProvider.NAME, question.getAuthor(), false);
+          if (userIdentity == null) {
+            LOG.warn("Failed to move questions : userIdentity = null");
```

Comment:
The message should be more meaningful, something like : "Question " + questionId + " cannot be moved because identity of the user " + question.getAuthor() + " does not exist".